SU-SWS/stanford_profile
# D8CORE-1201 use confirm_leave module

# READY FOR REVIEW

# Summary
- Added confirm_leave module

# Need Review By (Date)
- very soon

# Urgency
- high

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-1201-confirm-leave`
1. checkout https://github.com/SU-SWS/react_paragraphs/pull/8
1. `drush cim -y`
1. go to create or edit a node
1. edit any field anywhere
1. try to leave the page via back button or another link
1. validate you get a confirmation alert.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
